RFCs/0016-Loggers.md

```diff
@@ -0,0 +1,85 @@
+# 0016 Loggers
+
+## Summary
+This note outlines the proposed changes for:
+1. Enabling logger support in test platform for TranslationLayer clients.
```

Comment:
We support both protocol based and C# library based clients, right?

Reply:
This will allow developers to specify trx or xunit or nunit loggers from Test Explorer?

Reply:
1. Correct. Changes will enable logger support for both protocol based and C# library based clients. I will correct the RFC.

2. No, user can't specify loggers from test explorer. Loggers can be specified from command line and run settings.

Reply:
Doesn't VS allow specifying runsettings through `Test | Test Settings` menu? Do we block the flow anywhere if user specifies loggers in the runsettings there?

Reply:
Loggers are supported in run settings from Test settings menu also. 
Earlier, I thought the question was about some first class support for adding loggers directly from test explorer (and not run settings)

Reply:
Awesome!